Hazelnut/premake5.lua

```diff
@@ -27,6 +27,25 @@ project "Hazelnut"
 		"Hazel"
 	}
 
+	filter "system:linux"
+		links
+		{
+			"GLFW",
+			"Glad",
+			"ImGui",
+			"Xrandr",
+			"Xi",
+			"GLU",
+			"GL",
+			"X11",
+			"dl",
+			"pthread",
+			"stdc++fs"
+		}
```

Comment:
Do you really need to link against all of them? In my experience the following are enough but it may differ per distro. I'm running Linux Mint 20, which is Ubuntu based.
```suggestion
		links
		{
			"GLFW",
			"Glad",
			"ImGui",
			"X11",
			"dl",
			"pthread",
		}
```

Reply:
Exacly I use this code
```lua
LinkLibs = {
	"GLFW",
	"Glad",
	"ImGui",
	"Lua",
	"Box2D"
}
```
```lua
links (LinkLibs)

filter "system:linux"
		pic "on"
		systemversion "latest"
		staticruntime "on"
		links { "dl", "pthread" }
```
I think even X11 is not needed but I'm not sure.
Linux Mint 19.3